libprotonup/src/file.rs

```diff
@@ -23,6 +24,17 @@ fn path_result(path: &Path) -> String {
 }
 
 pub fn decompress(from_path: &Path, destination_path: &Path) -> Result<()> {
+    let path_str = from_path.as_os_str().to_string_lossy();
+
+    if path_str.ends_with("tar.gz") { decompress_gz(from_path, destination_path) }
+
+    else if path_str.ends_with("tar.xz") { decompress_xz(from_path, destination_path) }
+
+    else { println!("no decompress\nPath: {:?}", from_path); Ok(()) }
```

Comment:
Maybe this should return an error.
I don't think there is a way to download this without it being compressed.

Reply:
Sounds good, this is a leftover from trying to troubleshoot why the ends_with didn't appear to be working